src/book/book.rs

```diff
@@ -264,6 +264,10 @@ fn load_chapter<P: AsRef<Path>>(
             format!("Unable to read \"{}\" ({})", link.name, location.display())
         })?;
 
+        if content.as_bytes().starts_with(b"\xef\xbb\xbf") {
+            content = content[3..].to_string()
```

Comment:
Just to avoid the extra allocation, perhaps this could be `content.replace_range(..3, "");`